src/main/java/cz/student/zvedamrance/TaskManager.java

```diff
@@ -1,9 +1,119 @@
+        final String name = scanner.nextLine();
+        System.out.println("Please enter the date (format YYYY-MM-DD):");
+        final String date = scanner.nextLine();
+        System.out.println("Please enter if the task is done or not (true/false):");
+        final String isDone = scanner.nextLine();
+        tasks[tasks.length - 1] = new String[] {name, date, isDone};
     }
 
+    public static void remove() {
+        System.out.println("Please, enter the number of the task you would like to remove:");
+        int number = -1;
+        while (true) {
+            final String input = scanner.nextLine();
+            try{
+                number = Integer.parseInt(input);
+            } catch (NumberFormatException e) {
+            }
+            if (number >= 0 && number < tasks.length) {
+                break;
+            } else {
+                System.out.println(ConsoleColors.RED + "Please, enter a number between 0 and " + (tasks.length - 1) + "." + ConsoleColors.RESET);
+            }
+        }
+        tasks = ArrayUtils.remove(tasks, number);
+    }
+
+
+    private static void list() {
+        System.out.println(ConsoleColors.PURPLE + "Id:[Note, Date, IsDone]");
+        System.out.println(ConsoleColors.RESET + "-----------------");
+        for (int i= 0; i < tasks.length; i++) {
+            System.out.println( i + ":"  + Arrays.toString(tasks[i]));
+        }
+    }
+
+    public static void exit() {
+        saveTasks();
+        System.out.println(ConsoleColors.RED + "\nProgram terminated.");
+        scanner.close();
+        System.exit(0);
+    }
+    public static void saveTasks () {
+        try (Writer writer = new FileWriter(file)) {
+            for (int i = 0; i < tasks.length; i++) {
+                for (int j =0 ; j < 3; j++) {
+                    if (j < 2) {
+                        writer.write( tasks[i][j]+ ", ");
+                    } else {
+                        writer.write(tasks[i][j] + "\n");
+                    }
+                }
+            }
+        } catch (IOException e) {
+            System.err.println("Failed to save tasks into file.");
+            e.printStackTrace(System.err);
+            System.exit(1);
+        }
+    }
+    public static void loadTasks () {
+        final ArrayList<String> lines = new ArrayList<>();
+
+        try (Scanner scanner = new Scanner(file)) {
+            while (scanner.hasNextLine()) {
+                lines.add(scanner.nextLine());
+            }
+        } catch (IOException e) {
+            System.err.println("Failed to load tasks.csv");
+            e.printStackTrace(System.err);
+            System.exit(1);
+        }
+        tasks = new String[lines.size()][3];
+        for (int i = 0; i < lines.size(); i++) {
+            for (int j = 0; j < 3; j++) {
+                String [] temp = lines.get(i).split(",");
+                tasks[i][j] = temp [j].trim();
+            }
+        }
```

Comment:
IMPROVE: dle me by nacteni pole ukolu slo jeste zjednodusit.

```suggestion
        for (int i = 0; i < lines.size(); i++) {
            final var task = lines.get(i).split(",");
            tasks[i] = task;
        }
```

Reply:
opraveno